src/index.ts

```diff
@@ -203,3 +241,4 @@ export type RuneGameEvent =
   | { type: "GAME_OVER"; score: number; challengeNumber: number }
   | { type: "ERR"; errMsg: string }
   | { type: "SCORE"; score: number; challengeNumber: number }
+  | { type: "_INITIAL_OVERLAY_CLICK"}
```

Comment:
I do not think we have to prefix that with `_`. Could this be simply `INITAL_OVERLAY_CLICK`?

Reply:
This is an internal event that sdk users don't need to worry about. using _ indicates that :)

Reply:
I'm not sure this event is more "internal" than the other events? None of these events are used by game devs. They're only sent to the Rune app / browser :)

Reply:
Ah, I see. We can change it to not use _ then. 
The only thing is that they are "hackish" events, so wondering how to indicate that.

Reply:
Makes sense. Another way could be `BROWSER_INITIAL_OVERLAY_CLICK` and `BROWSER_GAME_LOAD_STARTED`. That way, any devs working on the Rune app would know that it's not relevant to them.

Anyway, not super important. Just wanted to clarify any potential confusion about who uses the events :)

Reply:
If we wanna change it, we should probably do it before rolling out to production to keep our life simple.

Reply:
FYI @mattbretl in case you have any thoughts on whether wanna use _ for events going forward :)

Reply:
The `BROWSER_` prefix sounds great to me. 👍

Reply:
I'll change it to BROWSER before releasing to production, thanks guys :)